_component_design/footer.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Footer
+parent: Navigation
+layout: component
+category: Components
+usage: The footer lives at the bottom of the page to help visitors find
```

Comment:
Add a `>` after `usage: ` and then add a soft return. It will look like this:
```yaml
usage: >
  The footer lives at the bottom of the page to help visitors find
  globally applicable, but non-primary navigation information. Use it on
  every page and every wrapped application. For customer applications,
  it should not exceed the articulated scope.
```

Alternatively, you could use a pipe (`|`). See https://stackoverflow.com/a/18708156 for details.